README.md

```diff
@@ -1,8 +1,108 @@
 # dcos-images
 
-The reference for DC/OS images and their prerequisites
-[DC/OS Platform Version Compatibility Matrix](https://docs.mesosphere.com/version-policy/#dcos-platform-version-compatibility-matrix)
+The reference for DC/OS images and how to build them
 
-Each dcos_images file should be a mapping of docker_version to region_ids to image_ids
+## Which DC/OS image should I use?
+Look for the dcos_images.yaml file in the directory path that matches your requirements.  
+Example: [centos/7.4/aws/DCOS-1.11.3/docker-1.13.1/dcos_images.yaml](https://github.com/dcos/dcos-images/blob/master/centos/7.4/aws/DCOS-1.11.3/docker-1.13.1/dcos_images.yaml).
 
-(work in progress)
+## How can I see a high-level view of what we support?
+1. ```git clone https://github.com/dcos/dcos-images.git ~/dcos-images```
+2. ```brew install tree``` or ```apt install tree```
+3. ```cd ~/dcos-images```
+4. ```tree```
+
+There's also a [DC/OS Platform Version Compatibility Matrix](https://docs.mesosphere.com/version-policy/#dcos-platform-version-compatibility-matrix)
+
+## How to build new DC/OS images
+All the files required to build new DC/OS images are in the yellow-colored boxes of this [diagram](flow-diagram.png).  
+If you're adding a new operating system, you'll need to create all these files. You should definitely use existing ones
+as a starting point.  
+Let's go over each one:
+
+- **base_images.json**  
+Contains images published directly by the OS providers (CoreOS, CentOS, etc.). You can usually find them on the
+provider's website. For example, you can find CoreOS AMIs [here](https://coreos.com/os/docs/latest/booting-on-ec2.html)
+
+- **install_dcos_prerequisites.sh**  
+Script that will run and be baked on top base images, which will form new images called a DC/OS images. 
+
+- **packer.json**  
+Packer configuration to build DC/OS images. Base images and install_dcos_prerequisites.sh are some of its inputs.
+
+- **desired_cluster_profile.tfvars**  
+Configuration for the terraform cluster that will be created using new DC/OS images. This cluster will be used to install
+and test DC/OS with the new images.
+
+- **publish_and_test_config.yaml**  
+Determines what is required for newly-built images to be commited back to a pull request by jenkins. Also determines
+what tests are ran for on a pull request. [See this example](https://github.com/dcos/dcos-images/blob/master/oracle-linux/7.4/aws/DCOS-1.11.3/docker-1.13.1/publish_and_test_config.yaml#L1).
+You can specify the field 'publish_dcos_images_after' with possible values: 'packer_build', 'dcos_installation',
+'integration_tests' and 'never'. For example, if I want jenkins to push the newly-built DC/OS AMIs to my pull request as
+soon as they are built by packer, I would specify 'packer_build'. If I want more confidence and want to make sure
```

Comment:
Do not use `I want` , `I` etc in the README.  Write the entire document in the 3rd person ( https://en.wikiversity.org/wiki/Talk:Technical_writing_style)

Reply:
@orsenthil What you linked specifically mentions not to write in the 3rd person. It says to use 2nd person.